prow/plugins/bugzilla/bugzilla.go

```diff
@@ -43,6 +43,9 @@ var (
 	refreshCommandMatch  = regexp.MustCompile(`(?mi)^/bugzilla refresh\s*$`)
 	qaAssignCommandMatch = regexp.MustCompile(`(?mi)^/bugzilla assign-qa\s*$`)
 	qaReviewCommandMatch = regexp.MustCompile(`(?mi)^/bugzilla cc-qa\s*$`)
+	cherrypickPRMatch    = regexp.MustCompile(`This is an automated cherry-pick of #([0-9]+)`)
```

Comment:
can we factor this out into a constant to share with the cherrypicker

Reply:
Not sure what we could share here? This is a regex and the cherrypicker plugin has an `Sprintf` string. Is there a way to combine an `Sprintf` string and regex?

Reply:
Maybe break out `This is an automated cherry-pick of ` as a constant? Or better write a unit test that ensures that the cherry-picker function to generate a title is always matching the regex here.